docker/mainnet/ethereum/ethereum.mainnet.conf

```diff
@@ -0,0 +1,59 @@
+[Eth]
+NetworkId = 1
+SyncMode = "fast"
+LightPeers = 100
+DatabaseCache = 768
+GasPrice = 18000000000
+EnablePreimageRecording = false
+
+[Eth.Ethash]
+CacheDir = "ethash"
+CachesInMem = 2
+CachesOnDisk = 3
+DatasetDir = "/root/.ethereum/ethash-dataset"
+DatasetsInMem = 1
+DatasetsOnDisk = 2
+PowMode = 0
+
+[Eth.TxPool]
+NoLocals = false
+Journal = "transactions.rlp"
+Rejournal = 3600000000000
+PriceLimit = 1
+PriceBump = 10
+AccountSlots = 16
+GlobalSlots = 4096
+AccountQueue = 64
+GlobalQueue = 1024
+Lifetime = 10800000000000
+
+[Eth.GPO]
+Blocks = 20
+Percentile = 60
+
+[Shh]
+MaxMessageSize = 1048576
+MinimumAcceptedPOW = 2e-01
+
+[Node]
+NoUSB = true
+DataDir = "/root/.ethereum"
+IPCPath = "/root/.ethereum/geth.ipc"
```

Comment:
Why do we need ipc enabled?

Reply:
It's default option generated by `geth dumpconfig`.

Reply:
Disabled IPC and WS